src/demo/models/guide.js

```diff
@@ -122,11 +121,10 @@ const guides = [
   },
   {
     id: 'fishvtrash-pond-recall',
-    text: `Switch between “fish” and “not fish” here.`,
+    text: `Click here to switch between objects labeled as “Fish” and “Not Fish”.`,
```

Comment:
i can't remember what we decided on here -- should it be "fish" / "not fish" (lowercase) or "Fish" / "Not Fish" (titlecase)

Reply:
Yes it should be lowercase except we literally refer to the Fish/Not Fish buttons in one place.

Reply:
fixed!